yaptide/routes/user_routes.py

```diff
@@ -17,8 +15,8 @@
 from yaptide.routes.utils.response_templates import yaptide_response, error_validation_response
 
 MAX_PAGE_SIZE = 100
-DEFAULT_PAGE_SIZE = 10
-DEFAULT_PAGE_IDX = 0
+DEFAULT_PAGE_SIZE = 6
+DEFAULT_PAGE_IDX = 1
```

Comment:
remove or use as default value when the request didn't provide any value.

Reply:
Done in 3ce3177